pex/bin/pex.py

```diff
@@ -500,9 +500,11 @@ def configure_clp_pex_environment(parser):
         callback=process_platform,
         help="The platform for which to build the PEX. This option can be passed multiple times "
         "to create a multi-platform pex. To use the platform corresponding to the current "
-        "interpreter you can pass `current`. To target any other platform you pass a string "
-        "composed of fields: <platform>-<python impl abbr>-<python version>-<abi>. These fields "
-        "stem from wheel name conventions as outlined in "
+        "interpreter you can pass `current`. If any distributions need to be built, use the "
+        "--python or --interpreter-constraint argument instead, providing the corresponding "
```

Comment:
Ok. I think explaining complex things in certain contexts like CLI help and error messages is a tarpit but I know others don't agree. This now tells half the truth. The whole truth is you can also pass `--resolve-local-platforms`.

Reply:
I think `--resolve-local-platforms` has exactly the right information and it is directly below `--platforms`. I will revert this change.